src/main/java/de/retest/web/WebData.java

```diff
@@ -0,0 +1,74 @@
+package de.retest.web;
+
+import java.awt.Rectangle;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class WebData {
+
+	private static final Logger logger = LoggerFactory.getLogger( WebData.class );
+
+	private final Map<String, Object> webData;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename field "webData" ([squid:S1700](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1700))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=42&issues=AWT5dkLxJ00hBq3sh87j&open=AWT5dkLxJ00hBq3sh87j)

[ISSUE_KEY]: AWT5dkLxJ00hBq3sh87j